source/_posts/2017-06-30-uefi-slash-edk-ii-corebootpkg-on-pc-engines-apu2.markdown

```diff
@@ -0,0 +1,335 @@
+---
+author: Piotr Król
+layout: post
+title: "UEFI/EDK II CorebootPayloadPkg on PC Engines APU2"
+date: 2017-06-30 22:14:55 +0200
+comments: true
+categories: coreboot uefi edk2 apu2
+---
+
+Recently we were reached by person interested in running CoreOS on APU2. CoreOS
+is very interesting system from security point of view it was created to
```

Comment:
@pietrushnic I feel like `it was created to support containers out of the box.` should be another sentence or at lest with dash (`-`) at the front of it